omnigibson/robots/two_wheel_robot.py

```diff
@@ -1,11 +1,11 @@
 from abc import abstractmethod
 import gym
 import numpy as np
+from telemoma.human_interface.teleop_core import TeleopAction 
```

Comment:
Don't import this here. Creates an unnecessary dependency. Remove the type hint it's used for.